packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -211,15 +211,15 @@ class CupertinoDatePicker extends StatefulWidget {
   /// the intervals set in [minimumDate], [maximumDate], [minimumYear], and
   /// [maximumYear].
   ///
-  /// [minimumDate] is the minimum [DateTime] that the picker can be scrolled to.
-  /// Null if there's no limit. In [CupertinoDatePickerMode.time] mode, if the
-  /// date part of [initialDateTime] is after that of the [minimumDate], [minimumDate]
-  /// has no effect.
+  /// [minimumDate] is the minimum selectable [DateTime] of the picker. In
+  /// [CupertinoDatePickerMode.time] mode, [minimumDate] should typically be on
+  /// the same date as [initialDateTime]. When set to null, the picker does not
```

Comment:
Is this right? it's a bit different from the original doc. What happens if it's not?

Reply:
Updated the doc to explain the suggestion. I believe they're roughly equivalent (has no effect if too early, invalid if too late)?